src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -171,14 +178,29 @@ private static ProcessModuleCollection GetModules(int processId, bool firstModul
 
                         module.ModuleName = new string(chars, 0, length);
 
-                        length = Interop.Kernel32.GetModuleFileNameEx(processHandle, moduleHandle, chars, chars.Length);
+                        while (true)
```

Comment:
You don't need a similar loop for module name above?  Or is there a maximum module name length that we need to always rent an array larger than, even in debug?

Reply:
Great question, I am going to verify that

Reply:
you were right, I've fixed that